src/main/java/com/boomi/flow/services/boomi/mdh/common/Entities.java

```diff
@@ -166,6 +169,25 @@ private static MObject createMatchesToProperty(Universe universe, Map<String, Ob
             }
         }
 
+        if (links == null) {
+            return  properties;
+        }
+        // this part is only for Golden Records
+        List<MObject> mObjectLinks = links.stream()
+                .map(Entities::createMObjectForLink)
+                .collect(Collectors.toList());
+
+        properties.add(new Property(GoldenRecordConstants.LINKS_FIELD, mObjectLinks));
         return properties;
     }
+
+    private static MObject createMObjectForLink(GoldenRecord.Link link) {
+        List<Property> linkProperties = new ArrayList<>();
+
+        linkProperties.add(new Property("Source", link.getSource()));
+        linkProperties.add(new Property("Entity ID", link.getEntityId()));
+        linkProperties.add(new Property("Established Date", link.getEstablishedDate().format(formatter)));
```

Comment:
I don't think you need to create a formatter here - you can just use `DateTimeFormatter.ISO_OFFSET_DATE_TIME`, or maybe even just pass `link.getEstablishedDate()` straight into the constructor of `Property`, as the string representation should be an ISO date time